docs/wsgi.rst

```diff
@@ -1,193 +0,0 @@
-------------------
-
-In the Apache config (or ``<VirtualHost>``) you will need the following:
-
-*Note that values may be slightly different.*
-
-::
-
-    DocumentRoot /path/to/kitsune/webroot/
-
-    <Directory "/path/to/kitsune/webroot/">
-        Options +FollowSymLinks
-    </Directory>
-
-    Alias /media/ "/path/to/kitsune/media/"
-    Alias /admin-media/ \
-        "/path/to/virtualenv/lib/python<version>/site-packages/django/django/contrib/admin/media/"
-
-    WSGISocketPrefix /var/run/wsgi
-
-    WSGIDaemonProcess kitsune processes=8 threads=1 \
-        maximum-requests=4000
-    WSGIProcessGroup kitsune
-
-    WSGIScriptAlias /k "/path/to/kitsune/wsgi/kitsune.wsgi"
-
-``WSGISocketPrefix``:
-    May or may not be necessary. It was for me.
-
-``WSGIDaemonProcess``:
-    ``processes`` should be set to the number of cores.
-    ``threads`` should probably be left at 1. ``maximum-requests`` is good at
-    between 4000 and 10000.
-
-``WSGIScriptAlias``:
-    Will make Kitsune accessible from ``http://domain/k``, and we use rewrites
-    in ``webroot/.htaccess`` to hide the ``/k``. This will change soon, and the
-    ``.htaccess`` file won't be necessary.
-
-The ``Alias`` directives let Kitsune access its CSS, JS, and images through
-Apache, reducing the load on Django.
-
-
-Configuration
--------------
-
-Most of our ``kitsune/settings.py`` is under version control, but can be
-overridden in a file called ``kitsune/settings_local.py``. You can see
-example settings in the :ref:`hacking-howto-chapter`.
-
-
-
-File Permissions
-================
-
-To upload files, the webserver needs write access to ``media/uploads`` and all
-its subdirectories. The directories we currently use are::
-
-    media/uploads
-    media/uploads/avatars
-    media/uploads/images
-    media/uploads/images/thumbnails
-    media/uploads/gallery/images
-    media/uploads/gallery/images/thumbnails
-    media/uploads/gallery/videos
-    media/uploads/gallery/videos/thumbnails
-
-``media/uploads`` and its subdirectories should never be added to version
-control, as they are installation-/content-specific.
-
-
-Product Details JSON
---------------------
-
-Some people have issues with ``django-mozilla-product-details`` and file
-permissions. The management command ``manage.py update_product_details`` writes
-a number of JSON files to disk, and the webserver then needs to read them.
-
-If you get file system errors from ``product_details``, make sure the files are
-readable by the webserver (should be by default) and the directory is readable
-and executable.
-
-By default, ``product_details`` stores the JSON files in::
-
-    path/to/virtualenv/lib/python<version>/site=packages/django-mozilla-product-details/product_details/json
-
-This is configurable. If you have multiple web servers, they should share this
-data. You can set the ``PROD_DETAILS_DIR`` variable in
-``kitsune/settings_local.py`` to a different path, for example on NFS.
-
-
-Debugging
-=========
```

Comment:
Maybe we could write something about debugging sumo in your local env. Both with the vscode and through docker without the editor.